db/schema.rb

```diff
@@ -34,15 +34,15 @@
   end
 
   create_table 'active_storage_variant_records', force: :cascade do |t|
-    t.integer 'blob_id', null: false
+    t.bigint 'blob_id', null: false
     t.string 'variation_digest', null: false
     t.index %w[blob_id variation_digest], name: 'index_active_storage_variant_records_uniqueness', unique: true
   end
 
   create_table 'diffs', force: :cascade do |t|
     t.integer 'layer_id', null: false
     t.integer 'seq'
-    t.json 'data', default: '{}', null: false
+    t.json 'data', default: '""{}""', null: false
```

Comment:
dont think we wanna keep these changes
```suggestion
    t.json 'data', default: '{}', null: false
```

Reply:
Oh ya good catch